hl/src/H5LT.c

```diff
@@ -787,11 +798,9 @@ H5LTopen_file_image(void *buf_ptr, size_t buf_size, unsigned flags)
         callbacks.udata = (void *)udata;
```

Comment:
I think the comment above: "corresponding to the first FAPL "should be changed to something like "the file driver holds a reference to udata, which is released when image_free is called"